### DISCUSSION TITLE
Separate page for Aliases?

### DISCUSSION THREAD

**Discussion by richb-hanover** [PRQL/prql#1255] [General]
This discussion is born of my minimal understanding of SQL. And since I don't really know the correct names, I just "look at the pictures". 

That is, I stumble through the Book, looking at examples for something that looks as if it'll do what I want (and with PRQL, mostly it does !!)

Perhaps there could be discussion (a page in the Book?) of Aliases because I didn't entirely twig to everything they could do for me... Examples I'd like to see:

```
# Set an alias for a table name
from e = employees
select e.first_name

# Set an alias for each table in a join
from l = employees
join r = departments [l.dept_ID == r.dept_id]
select [l.last_name, r.dept_name]

# Give a column an alias as its display name
from employees
select fullname = f"{first_name} {last_name}"
```
... what else?...

Thanks.

**Comment by max-sixty** [comment]
Yes I think this is a great idea!

**Comment by richb-hanover** [comment]
Follow-up to this discussion: I created PR #1267 to add links to the page that describes each feature. We still need pages/links for:

- Assignments & Aliases
- Comparisons
- Annotations
